include/mbedtls/ecdsa.h

```diff
@@ -171,31 +206,40 @@ int mbedtls_ecdsa_write_signature( mbedtls_ecdsa_context *ctx, mbedtls_md_type_t
 #define MBEDTLS_DEPRECATED
 #endif
 /**
- * \brief           Compute ECDSA signature and write it to buffer,
- *                  serialized as defined in RFC 4492 page 20.
- *                  Deterministic version, RFC 6979.
- *                  (Not thread-safe to use same context in multiple threads)
+ * \brief   This function computes an ECDSA signature and writes it to a buffer,
+ *          serialized as defined in <em>RFC-4492: Elliptic Curve Cryptography 
+ *          (ECC) Cipher Suites for Transport Layer Security (TLS)</em>.
+ *
+ *          The deterministic version, defined in <em>RFC-6979: 
+ *          Deterministic Usage of the Digital Signature Algorithm (DSA) and 
+ *          Elliptic Curve Digital Signature Algorithm (ECDSA)</em> is used
+ *          if #MBEDTLS_ECDSA_DETERMINISTIC is defined.
+ *          It is not thread-safe to use same context in multiple threads.
  *
  * \deprecated      Superseded by mbedtls_ecdsa_write_signature() in 2.0.0
  *
- * \param ctx       ECDSA context
- * \param hash      Message hash
- * \param hlen      Length of hash
- * \param sig       Buffer that will hold the signature
- * \param slen      Length of the signature written
- * \param md_alg    MD algorithm used to hash the message
+ * \param ctx       The ECDSA context.
+ * \param hash      The Message hash.
+ * \param hlen      The length of the hash.
+ * \param sig       The buffer that holds the signature.
+ * \param slen      The length of the signature written.
+ * \param md_alg    The MD algorithm used to hash the message.
  *
- * \note            The "sig" buffer must be at least as large as twice the
- *                  size of the curve used, plus 9 (eg. 73 bytes if a 256-bit
- *                  curve is used). MBEDTLS_ECDSA_MAX_LEN is always safe.
+ * \note            The \p sig buffer must be at least twice as large as the
+ *                  size of the curve used, plus 9. For example, 73 Bytes if a 
+ *                  256-bit curve is used. #MBEDTLS_ECDSA_MAX_LEN is always safe.
  *
- * \note            If the bitlength of the message hash is larger than the
+ * \note            If the bitlength of the message hash is larger than the 
```

Comment:
Blocker: Please remove the trailing whitespace.

Reply:
done